EXT_BeamPhysics.md

```diff
@@ -0,0 +1,252 @@
+Extension to the openPMD Standard for Describing Particle Beams and X-rays
+==========================================================================
+
+Version 2.0.0
```

Comment:
Let us not write the version here again, since the standard otherwise needs to be updated at several positions at once when bumping it.

Instead, write prominently the extension name:
```suggestion
openPMD extension name: `BeamPhysics`
```

Reply:
@ax3l 
At this point in time we are talking about having to modify 4 files each time the version is changed. The other side of the argument is that if someone stores a standards file outside of GIT then later on it will be easy to see which version the file is for. This is not a big deal for me so if you do think it is better then I will go along and remove it. What do you think?

Reply:
Yes, let's keep it only in the main file for now :)

Reply:
Done